section1/reflection.md

```diff
@@ -1,19 +1,38 @@
+- I would explain this to a five year old simply as a piece of something else that can be changed, without changing the entirety of the bigger picture. For example, the best way to explain this to a child is that variables are like toy boxes. The toy box holds an amount of toys, and that amount can be whatever we want it to be (a value). No matter what that value is, the toy box represents that amount of toy which can change (like a variable).
 
-7. In your own words, explain what concatenation is.
+### 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
+- Amazon is a website I use often, and variables used would probably be facets such as search engine verbiage, payment information, quantity of the product desired, membership subscriptions, etc.
+- In this case, the words used in the search engine will likely hold a string data type, similar to the payment/billing information portion of checkout. However, the membership subscription verification will likely use a bollean data type (verifying yes/no, true/false if the membership exists and is active). Additionally, the quantity of the product desired will most likely hold a strictly numeric data type.
 
-8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
+### 7. In your own words, explain what concatenation is.
+- Concatenation is a programming term used to describe joining strings together within JavaScript. It can be used to join together particular values utilizing several variables in order to create full sentences and interactions with the user of a particular website, app, etc.
 
-9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+### 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
+- A perfect example of an app/website that I think the developers used concatenation would be the Target app. Anytime you login to the app, you can see where the top of the page is greeting the user with their personalized name (something that was inputted when the user created their profile within the app). For example, it says `"Hello, Maia!"` using the user's first name being the variable's value in this case. I would be willing to guess that this would also apply to "recently purchased items" portions of the user interface as well as "purchase again?" -- where the developers would simply pull a log from some sort of link from a previously confirmed purchase that itemizes the last shopping purchase you made within the site.
+
+### 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+- Are concatenations and interpolations data types, or are they terms that are separate from data types? If so, what is the difference between the two?
```

Comment:
Great question!

No, they aren't technically data types themselves. I would think of them as a tool or structure JS provides. The RESULT in a String, though. Even if you concatenate 2 Numbers, the result is a String.